content/blog/2021/06/camel-quarkus-release-2.0.0/index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Camel Quarkus 2.0.0 Released"
+date: 2021-06-30
+authors: ["ppalaga"]
+categories: ["Releases", "Camel Quarkus"]
+preview: "Camel Quarkus 2.0.0 Released"
+summary: "Camel Quarkus 2.0.0 brings Quarkus 2.0.0.Final and Camel 3.11.0"
+---
+
+<sub><sup>Image by <a href="https://www.flickr.com/photos/lanzen/5984113332">Anders Lanzen</a> <a href="https://creativecommons.org/licenses/by-nc-sa/2.0">CC BY-NC-SA 2.0</a></sup></sub>
+
+We are pleased to announce the release 2.0.0 of Camel Quarkus.
+It brings Quarkus 2.0, Camel 3.11, improved test coverage and 13 new or promoted extensions.
+Many thanks to all contributors and issue reporters!
+
+## Quarkus 2.0.0.Final
+
+Like in Camel 3.11, in Quarkus 2.0, Vert.x and MicroProfile were upgraded to their newest major versions 4.
+Further, in Quarkus 2.0, Java 8 support was dropped permanently and Java 11 is the minimal supported version.
+21.1 is the recommended version of GraalVM.
+
+When it comes to developer experience, Quarkus 2.0 introduces continuous testing.
+Quarkus dev mode (a.k.a. `mvn quarkus:dev`) exists since the very beginnings of Quarkus.
+It not only starts your application so that you can test it manually,
+but it also keeps watching for changes in your workspace, recompiling the modified classes as needed
+and reloading the running application.
+Now with continuous testing, your tests are executed as you change your application code.
+You have to press `r` to enable it.
+You may want to watch [this video](https://www.youtube.com/watch?v=rUyiTzbezjw) to learn more about the new continuous testing feature.
+
+Please refer to [Quarkus 2.0.0.Final announcement](https://quarkus.io/blog/quarkus-2-0-0-final-released/) for more details.
```

Comment:
Its 404 today, but I guess it should be fine.

Reply:
Yeah, it is blocked by this one: https://github.com/quarkusio/quarkusio.github.io/pull/1037